File: python/tools/replace_in_list.py
def replace_in_list(sin, *args):
    sstr = args[0::2]  # Extract search strings (odd indices)
    rstr = args[1::2]  # Extract replace values (even indices)

    for j in range(len(sin)):
        if not isinstance(sin[j], str):
            continue
        for k in range(len(sstr)):
            if sin[j] == sstr[k]:
                sin[j] = rstr[k]
                break
    
    return sin

File: MATLAB/tools/replaceinlist.m
function sin=replaceinlist(sin,varargin)
sstr=varargin(1:2:end);
rstr=varargin(2:2:end);
for l=1:length(sin)
    for k=1:length(sstr)
        if strcmp(sin{l},sstr{k})
            sin{l}=rstr{k};
        end
    end
end
end